assets/sass/components/dashboard/wizard/_googlesitekit-wizard-progress.scss

```diff
@@ -37,4 +37,28 @@
 			padding: $grid-gap-desktop 0 0 0;
 		}
 	}
```

Comment:
The rules in this file almost all apply to the legacy setup flow which is only visible when _not_ using the proxy (only relevant for development or staging sites). The `SetupProxy` component only uses `.googlesitekit-wizard-progress`, which is some legacy remainder. We don't need to change that at this point, but let's rather decouple the new classes from that and make them based on `.googlesitekit-setup__*` (and put them into `_googlesitekit-setup.scss`).

Reply:
Sounds good @felixarntz . I moved the new class names. Can you check again?